packages/combo-box/src/vaadin-combo-box-data-provider-mixin.d.ts

```diff
@@ -58,4 +51,8 @@ interface ComboBoxDataProviderMixin<TItem> {
   clearCache(): void;
 }
 
-export { ComboBoxDataProviderMixin, ComboBoxDataProviderMixinConstructor };
+export declare function ComboBoxDataProviderMixin<TItem, T extends Constructor<HTMLElement>>(
+  base: T
+): T &
+  Constructor<ComboBoxDataProviderHost<TItem>> &
+  Pick<typeof ComboBoxDataProviderHost, keyof typeof ComboBoxDataProviderHost>;
```

Comment:
I could not figure out why the `Pick` is necessary, it seems the resulting type already receives all members from the host class through the `Constructor<...Host>` type.

Reply:
Yep, doesn't seem to be needed. It was a copy-paste from Lion, will remove it to keep things simple.

Reply:
Removed.